docs/extensibility/theme-support.md

```diff
@@ -91,3 +91,63 @@ add_theme_support( 'disable-custom-colors' );
 ```
 
 This flag will make sure users are only able to choose colors from the `editor-color-palette` the theme provided or from the editor default colors if the theme did not provide one.
+
+## Editor styles
+
+A theme can provide a stylesheet to the editor itself, to change colors, fonts, and any aspect of the editor.
```

Comment:
The comma usage here confused me, I think the first one is a semicolon really. How about:

<pre>A theme can provide a stylesheet that will style the editor. You can use this to change colors, fonts, and any other visual aspect of the editor.</pre>